src/workerd/api/worker-rpc.h

```diff
@@ -47,7 +57,76 @@ class WorkerRpc : public Fetcher, public jsg::NamedIntercept {
   }
 };
 
-#define EW_WORKER_RPC_ISOLATE_TYPES  \
-  api::WorkerRpc
+// The capability that lets us call remote methods over RPC.
+// The client capability is dropped after each callRequest().
+class JsRpcTargetImpl final : public rpc::JsRpcTarget::Server {
```

Comment:
nit: Can we move this impl to the source file?

Reply:
https://github.com/cloudflare/workerd/compare/0e93bf986d59f2e5cadda2b80d10aee5734f5ce9..e1f09aaa04e65125995e646e2dfc46879e34e5a1

Note that in moving it into the source file I've also moved the `JsRpcTargetImpl::call()` definition into the class.